docker/docker/src/main/java/org/arquillian/cube/docker/impl/docker/compose/ComposeBuilder.java

```diff
@@ -51,13 +56,24 @@ public DockerCompositions build(Map<String, Object> dockerComposeContainerDefini
                 } else {
                     final Map<String, Object> serviceDefinition = asMap(servicesDefinition, key);
                     if (serviceDefinition.containsKey(NETWORKS)) {
-                        final Collection<String> serviceNetworks = asListOfString(serviceDefinition, NETWORKS);
-                        if (!serviceNetworks.isEmpty()) {
-                            cubeContainer.setNetworks(new HashSet<>(serviceNetworks));
-                            cubeContainer.setNetworkMode(serviceNetworks.iterator().next());
-                        } else {
-                            throw new IllegalArgumentException("Networks not mentioned under services networks section.");
-                        }
+                        Collection<String> serviceNetworks;
+                        Map<String, Object> serviceNetworksConfig = null;
+                            if (serviceDefinition.get(NETWORKS) instanceof ArrayList) {
```

Comment:
Currently after parsing networks from serviceDefinition we are getting Only ArrayList or Map. Other types of lists/collections are not supported. Previously it was supported only for List.